sbws/lib/v3bwfile.py

```diff
@@ -278,6 +280,11 @@ def bw_strv110(self):
         """Return Bandwidth Line string following spec v1.1.0."""
         bw_line_str = BW_KEYVALUE_SEP_V110.join(
                         self.bw_keyvalue_v110str_ls) + LINE_SEP
+        if len(bw_line_str) > BW_LINE_SIZE:
+            # if this is the case, probably there are too many KeyValues,
+            # or the limit needs to be changed in Tor
```

Comment:
How close are we to the limit?

Let's change the limit in Tor 0.3.5, so that the new limit is in production by the time we need it:
https://trac.torproject.org/projects/tor/ticket/26223